security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Menu/Menu.xml

```diff
@@ -0,0 +1,5 @@
+<menu>
+  <Services>
+    <ClamAV cssClass="fa fa-ambulance" url="/ui/clamav/general/index" />
```

Comment:
fa-user-md, fa-stethoscope may both fit better as they are both related to analysis while ambulance is "transport" and may fit more the the c-icap av_scan (transports the file to clamav).